README.md

```diff
@@ -1,2 +1,56 @@
 # Ecommerce
 A Django project to build an online bookstore
+
+# Set up
+For this project to work properly, before starting the server, you will need to first create a virtual environment and then download all the assets listed in requirements.txt. Afterwards you'll need to make and run some migrations for Django to build a database (by default in SQLite3), and then enable stripe for payments to work.
+
+- **Creating a virtual environment**
+
+   In your console, navigate to the folder where you installed the project. Run the folowing command:
+   
+   `python -m venv <path_to_venv>`
+   
+   `path_to_venv` can be either a relative or absolute path. Make sure NOT to use the angled brackets when writing the path. For example, `./venv` will create a new folder for your virtual environment on your current directory called venv. Please note that it may take a few minutes for the virtual envronment to be created, depending on how good the computer running the command is.
+   
+- **Running the virtual environment**
+
+   Now that the venv is installed, you should see a folder called `Scripts` with an activation script inside. You can activate the venv by typing `./<path_to_venv>/Scripts/activate` in your console.
+
+- **Download the required assets from requirements.txt**
+
+  In requirements.txt you'll find a list of required assets and libraries that need to be downoaded first in order for the project to work correctly. You can install everything in that file to your current working environment (in this case the venv) by going to the console and writing `pip install -r requirements.txt`
+
+- **Create migrations**
+
+  Django uses what's called *models* to create a single, definitive source of information about your data. They contain the essential fields and behaviors of the data you’re storing, and generally map to a single database table.
+
+  This project already contains the finished models that it needs, however you first need to create the migrations required to create the database in the first place. For that, go to your console and write `python -m manage.py makemigrations`. This will create the migrations required for a database to be created and for the models to map to said database. By default, the database created is an SQLite3 database.
+
+- **Run the migrations**
```

Comment:
ideally, the project should support both dev mode and production mode. That is, start with either sqlite database or link to an SQL service (such as postgreSQL). There should be a database section in your README to explain this. See the other comment about the settings file on how to do this